app/src/androidTest/java/com/github/fribourgsdp/radio/PlayerGameHandlerTest.kt

```diff
@@ -256,7 +256,7 @@ class PlayerGameHandlerTest {
         handler.onPick(song)
 
         // Wait for the task of the database to execute
-        Thread.sleep(1)
+        Thread.sleep(100)
```

Comment:
Why did you need to extend to sleeping time?

Reply:
The test failed several times when I pushed, so I added time to make it pass every time. I notified Victor so that he could look into optimising this.